activejob/test/cases/argument_serialization_test.rb

```diff
@@ -170,7 +170,7 @@ class ClassArgument; end
   end
 
   test "allows for keyword arguments" do
-    KwargsJob.perform_later(argument: 2)
+    KwargsJob.perform_now(argument: 2)
```

Comment:
This change is to excercise `perform_now` with keyword arguments, `perform_later` is already tested by `MultipleKwargsJob`.

https://github.com/rails/rails/blob/d72f940d1612aa8b905214c06115db48af41e8ec/activejob/test/cases/test_helper_test.rb#L561-L570